packages/event-handler/src/rest/BaseRouter.ts

```diff
@@ -133,11 +139,49 @@ abstract class BaseRouter {
     };
   }
 
-  public abstract resolve(
+  public async resolve(
```

Comment:
Now that this function has an implementation, there are no abstract methos remaining. Should `BaseRouter` still be an abstract class?

Reply:
If there's no longer a reason for it to be abstract, I'm ok to do away with it.